util/topgen/strong_random.py

```diff
@@ -44,6 +46,17 @@ def load(self, input_file):
                     sys.exit(1)
         self.buffername = input_file
 
+    def generate_from_seed(self, buffer_size, seed):
+        """Load entropy buffer from an external file.
```

Comment:
stringdoc needs to be updated since this function doesn't load entropy from an external file. In the spirit of `strong_random` I wonder if this should be called `unsafe_generate_from_seed()`

Reply:
Fixed the docstring and renamed `generate_from_seed` to `unsafe_generate_from_seed()`.